python/scannerpy/database.py

```diff
@@ -73,7 +75,9 @@ def __init__(self, config_path=None, config=None):
         self._db = self._bindings.Database(
             self.config.storage_config,
             self._db_path,
-            self._master_address)
+            self._master_address,
```

Comment:
`self._master_address` and `self._worker_address` probably don't need to be variables if they're only used once.